packages/client/pages/post-job/index.tsx

```diff
@@ -68,7 +68,13 @@ export default function CreateProject() {
 
   //Sends data to database (sent to JobSummary as props)
   const createJob = async (e: React.FormEvent) => {
+    let user = await supabase.from('users').select('id');
     e.preventDefault();
+
+    const { data, error } = await supabase.from('jobs').upsert({
+      ...formData,
+      created_by: user['data']![0].id,
+    });
```

Comment:
Just realized you can omit making queries to get the current user and in the current query is also not correct because it will always set the first user record from the DB.

I think we need to use Context API and persist the wallet address of the user upon login, then call `useContext` and attach the `AuthContext` (just assuming the auth context exists)

```ts
const authContext = useContext(AuthContext);

const createJob = async (e: React.FormEvent) => {
  e.preventDefault();

  const { data, error } = await supabase.from("users").upsert({
    ...formData,
    created_by: authContext.walletAddress // or authContext.address whichever is written,
   })
}
```

Reply:
But I think we can use the `useEthers` hook for this to grab the user's wallet address